PendingReleaseNotes.md

```diff
@@ -4,6 +4,8 @@
 
 * The mds liveness and startup probes are now configured by the filesystem CR instead of the cluster CR. To apply the mds probes, they need to be specified in the filesystem CR. See the [filesystem CR doc](Documentation/ceph-filesystem-crd.md#metadata-server-settings) for more details. See #9550
 
+* Remove `caBundle` value from webhook config which is not required anymore.
```

Comment:
- This is a breaking change? What needs to be done to handle it? Will we need any instructions in the upgrade guide?
- Pending release notes are best in the same commit as the change, instead of a separate commit

Reply:
since, this is not breaking change, I have removed entry from releaseNotes